internal/hcsoci/create.go

```diff
@@ -109,9 +114,9 @@ func CreateContainer(ctx context.Context, createOptions *CreateOptions) (_ cow.C
 	if coi.HostingSystem != nil {
 		n := coi.HostingSystem.ContainerCounter()
 		if coi.Spec.Linux != nil {
-			resources.containerRootInUVM = "/run/gcs/c/" + strconv.FormatUint(n, 16)
+			resources.containerRootInUVM = fmt.Sprintf(lcowRootInUVM, createOptions.ID)
 		} else {
-			resources.containerRootInUVM = `C:\c\` + strconv.FormatUint(n, 16)
+			resources.containerRootInUVM = fmt.Sprintf(wcowRootInUVM, strconv.FormatUint(n, 16))
```

Comment:
On line 177 where we pass the `rootfs` for the OCI bundle we need this to include the `rootfs` folder. Or else do it in that method but the bundle path should include `root\rootfs`

Reply:
Oh sorry. We actually want the "rootfs" to be: `<rootInUvm>/bundle/rootfs`. Then in `/bundle` is where we write the config.json etc. But `/bundle/rootfs` is where we issue the CombineLayers